### PR TITLE
Added note to heatmap collector code about integrated solution in Tag manager for PPAS version >= 16.8.0

### PR DESCRIPTION
**Pull request by rezik007**
Why? https://piwikpro.slack.com/archives/C9L12C8HX/p1640075331131000

Preview:
Zrzut ekranu 2021-12-21 o 13 43 59

### CODE REVIEW

File: templates.js [hunk 1]
```diff
@@ -154,6 +154,7 @@ window._paq.push(['trackEvent', 'UX Research', 'Excessive Scroll', lastKnownPosi
     {
       id: 'heatmapClicks',
       name: 'Heatmap clicks collector',
+      beforeDescriptionNote: '<b>IMPORTANT: If your PPAS version is higher than 16.8.0 you should migrate to integrated, backward compatible solution which is <a target="_blank" href="https://help.piwik.pro/support/tag-manager/heatmap-clickmap-tag/">Heatmap & clickmap tag</a></b>',
```
Comment (zawias-pro) [inline templates.js:157]: I'm not sure whether we should recommend the migration. For sure we should recommend using the tag template for new setups.

**Review by zawias-pro — COMMENTED** [review]
looks nice, can we add a note about html5 videos too?

File: templates.js (hunk 1 above)
Comment (rezik007) [reply templates.js:157]: My purpose was only to engage people to use new tag solution if they find that code first. And in the end we should deprecate that code to not maintain them both.

**Comment by rezik007** [comment]
> looks nice, can we add a note about html5 videos too?

Added as "consider to use" but I'm still not conviced if we should also try to migrate hmmm 🤔  I'm not sure if tag covers all custom cases also, like for heatmaps it do (expect filtering given classnames but lets call it very margin case) .

**Comment by zawias-pro** [comment]
@rezik007 it does, the only exception is the number of decimal places for timestamps (e.g. user paused at 00:05.334) but we found it not so useful - we weren't able to find a case for any user to want to track split seconds.

**Review by zawias-pro — APPROVED** [review]

**Pull request merged**